README.md

```diff
@@ -5,30 +5,74 @@ This is the repository of the referee box for the RoCKIn competition (http://roc
 
-Additionally, the following debian packages need to be installed:
+### Officially Supported Setup: Ubuntu 14.04, Boost 1.54
 
-    sudo apt-get install libssl-dev libelf-dev mongodb-dev mongodb-clients mongodb libzmq3-dev
-  
-## Compiling the RoCKIn@Work Referee Box
-### Cloning the Git repository:
+1. The CFH is based off LLSF refbox, and uses [clips](http://clipsrules.sourceforge.net/). Add [Tim's PPA](https://launchpad.net/~timn/+archive/ubuntu/clips) for clips:
+      
+        sudo add-apt-repository ppa:timn/clips
+    (Note: This PPA is maintained by the developers of the LLSF referee box. And only works for Ubuntu 12.04, 12.10 and 14.04. For others, see Alternative Setup.)
+    
+2. Update apt:
+        
+        sudo apt-get update
 
-    git clone https://github.com/rockin-robot-challenge/at_work_central_factory_hub.git
-    cd at_work_central_factory_hub
-    git checkout rockin
+3. Install the dependencies for both LLSFRB and CFH
+        
+        sudo apt-get install libmodbus-dev libclips-dev clips libclipsmm-dev \
+                             protobuf-compiler libprotobuf-dev libprotoc-dev \
+                             boost1.54-all-dev libmodbus-dev \
+                             libglibmm-2.4-dev libgtkmm-3.0-dev \
+                             libncursesw5-dev libyaml-cpp-dev libavahi-client-dev git \
+                             libssl-dev libelf-dev mongodb-dev mongodb-clients \
+                             mongodb libzmq3-dev
+    (Note: Boost 1.54 is specified to avoid causing apt-get broken package problems with ROS. If you are using another version of Boost see Alternative Setup.)
 
+4. Clone the CFH repo
+        
+        git clone https://github.com/rockin-robot-challenge/at_work_central_factory_hub.git
 
-### Compiling the refbox:
+5. Enter the CFH repo
+        
+        cd at_work_central_factory_hub
 
-    make
+7. Make the CFH project
+        
+        make
 
-    
-### Run the RoCKIn@Work Referee Box
+8. Go to Configuration Section before running the CFH.
 
-    ./bin/llsf-refbox
+
+## Configuration
+
+Full configuration details and explainations are found in the PDF documentation (https://github.com/rockin-robot-challenge/at_work_central_factory_hub_doc). 
+
+Only a few common options are outlined here.
+
+1. Configure the CFH with your editor
     
+        editor ./cfg/config.yaml
+
+    For testing the following changes are common:
+    - Change the CFH IP address in section "shell", this should be where you're running the CFH. Other machines communicating with the CFH need this value set correctly.
+        
+            refbox-host: !ipv4 localhost
+    - disable mongodb, all messages will be logged and in early developement this may not be desired:
```

Comment:
Well, disabling MongoDB results in no messages being logged (not all messages being logged).